packages/chart/src/components/LinearChart.jsx

```diff
@@ -91,6 +100,18 @@ export default function LinearChart() {
       }
     }
 
+    // DEV-3263 - If Confidence Intervals in data, then need to account for increased height in max for YScale
+    if (config.visualizationType === 'Bar' || config.visualizationType === 'Combo') {
+      let ciYMax = 0
+      if (config.hasOwnProperty('confidenceKeys')) {
+        let upperCIValues = data.map(function (d) {
```

Comment:
You don't need to change anything, but arrow functions can help with writing concise code and provide implicit returns:

 `data.map( (d) => d[config.confidenceKeys.upper])`